packages/preferences/src/common/json-validator.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+export const JsonValidator = Symbol("JsonValidator");
+
+export interface JsonValidator {
```

Comment:
You're right, I was thinking there might be other more specific things we want to do with the validator but yes this should be kept simple instead.